float32 bldc
float32 green_1
float32 green_2
float32 yellow_1
float32 yellow_2
float32 yellow_3